docs/user-guide/dream/dream-visualize-absorption.ipynb

```diff
@@ -0,0 +1,140 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "0",
+   "metadata": {},
+   "source": [
+    "# DREAM instrument view\n",
+    "\n",
+    "This notebook is a simple example of how to use the instrument view for the DREAM instrument.\n",
+    "\n",
+    "- The DREAM-specific instrument view is capable of slicing the data with a slider widget along a dimension (e.g. `tof`) by using the `dim` argument.\n",
+    "- There are also checkboxes to hide/show the different elements that make up the DREAM detectors.\n",
+    "\n",
+    "We begin with relevant imports.\n",
+    "We will be using tutorial data downloaded with `pooch`.\n",
+    "If you get an error about a missing module `pooch`, you can install it with `!pip install pooch`:"
```

Comment:
I think the text in here needs updating? (seems it was copied from the instrument view notebook)